pyproject.toml

```diff
@@ -62,6 +62,8 @@ development = [
   "pytest==8.3.3",
   "pytest-randomly==3.15.0",
   "syrupy==4.7.1",
+  "factory-boy==3.3.0",
+  "Faker==26.0.0",
```

Comment:
Have you check licenses and CVEs for those libraries?

Reply:
MIT licenses and `snyk/requirements.txt` checks are passing (after adding these two deps). 
Any other checks I need to do/look for other than the snyk scan?

Reply:
We should be good 👌